lib/python/qmk/cli/format/json.py

```diff
@@ -61,5 +62,11 @@ def format_json(cli):
 
                 json_file['layers'][layer_num] = current_layer
 
+    output = json.dumps(json_file, cls=json_encoder, sort_keys=True)
+
+    if cli.args.inplace:
+        with open(cli.args.json_file, 'w+') as outfile:
+            outfile.write(output)
+
     # Display the results
-    print(json.dumps(json_file, cls=json_encoder, sort_keys=True))
+    print(output)
```

Comment:
Looks like with `--inplace` it would both write the formatted JSON back into the file and print the same JSON to stdout; is this intended and useful?

Reply:
Yes - I didn't want to remove the stdout output as to maintain as much parity with existing functionality as possible.
If you think it's better to be removed, it's a trivial change to make.

In terms of usefulness - I suppose it could be useful as a gut-check for someone to look at after the command runs, but they can also just do a `git diff` or open the formatted info.json.

Happy to do whichever the QMK team wants.

Reply:
imo, returning something like `qmk format-c` is doing by default https://github.com/qmk/qmk_firmware/blob/develop/lib/python/qmk/cli/format/c.py#L60 would probably be preferred to standardize on output and function . maybe do a `-v` or `-p` option to verbose/output in terminal. Up to QMK team for sure, thanks for the PR :+1:

Reply:
When I looked again - I realized that the format-c command seems to run in-place and modify the file by default. Setting --dry-run causes the output to be print to stdout.

Given format-json already has different default behavior, and changing it to run in-place without printing could break user expectations - I think adding the --print flag is still the right call.